src/auth/crud.py

```diff
@@ -17,6 +17,14 @@ async def create_user_session(db_session: AsyncSession, session_id: str, computi
     if existing_user_session:
         existing_user_session.issue_time = datetime.now()
         existing_user_session.session_id = session_id
+        query = sqlalchemy.select(models.User).where(models.User.computing_id == computing_id)
+        existing_user = (await db_session.scalars(query)).first()
+        if existing_user is None:
+            # @geb; throw?
```

Comment:
let's create a user row for them (like line 39), but log a warning for this case (since it should never occur unless something went wrong) using
```py
import logging
...
_logger = logging.getLogger(__name__)
...
# something like this
_logger.warning("Something is broken, aaaaaaaaaa")
```

`__name__` is the name of the current python module, so we can track it down more easily.

Reply:
cool done!